HomeWork_13/HW_10_BookStore_Reference/Printable.java

```diff
@@ -0,0 +1,28 @@
+package HomeWork_13.HW_10_BookStore_Reference;
+
+public interface Printable {
+    // Метод print, который должны реализовать классы, реализующие интерфейс Printable
+    void print();
+
+    // Дефолтный метод для печати массива объектов типа Printable (для книг)
+    default void printArrayBooks(Printable[] prints) {
```

Comment:
Such a method should be as it was before. Same for the method below.

Rewrite in Books and Magazine classes using stream API